### PR TITLE
[PropertyTask] Added type hints

### PR DESCRIPTION
**Pull request by siad007**
Related to #1201

### CODE REVIEW

File: classes/phing/tasks/system/PropertyTask.php [hunk 1]
```diff
@@ -120,7 +120,7 @@ public function getName()
      *
      * @param mixed $value Value of name, all scalars allowed
```
Comment (mimmi20) [inline classes/phing/tasks/system/PropertyTask.php]: Could you update the docblocks?

File: classes/phing/tasks/system/PropertyTask.php [hunk 2]
```diff
@@ -102,9 +102,9 @@ public function __construct(FileParserFactoryInterface $fileParserFactory = null
      *
      * @param $name
```
Comment (mimmi20) [inline classes/phing/tasks/system/PropertyTask.php]: Please add the type here.

File: classes/phing/tasks/system/PropertyTask.php [hunk 3]
```diff
@@ -132,7 +132,7 @@ public function setValue($value)
      * @internal param string $values
```
Comment (mimmi20) [inline classes/phing/tasks/system/PropertyTask.php]: Please update this docblock

File: classes/phing/tasks/system/PropertyTask.php [hunk 4]
```diff
@@ -257,9 +257,9 @@ public function getUserProperty()
     /**
      * @param $v
```
Comment (mimmi20) [inline classes/phing/tasks/system/PropertyTask.php]: Please add the type here.

**Comment by siad007** [comment]
@mimmi20  thx for your review, but your requests are actual not the scope of this  bug fix. Please open a new ticket or make a contribution.

**Pull request merged**